bot/code_review_bot/tasks/infer.py

```diff
@@ -48,13 +49,10 @@ def __init__(self, analyzer, entry, revision):
             nb_lines=1,
             check=entry["bug_type"],
             column=entry["column"],
-            level=kind,
+            level=Level(kind.lower()),
```

Comment:
Infer too for now should be always warnings

Reply:
Done.